common/compose/src/commonMain/kotlin/app/deckbox/common/compose/widgets/builder/CardBuilder.kt

```diff
@@ -105,8 +104,10 @@ fun CardBuilder(
     }
   }
 
+
   LaunchedEffect(isBottomSheetCollapsed) {
     if (isBottomSheetCollapsed) {
+      keyBoardController?.hide()
```

Comment:
Is this necessary? I would think that `focusManager.clearFocus()` + the field going out of composition would dismiss the keyboard. This is true as I test these features on my device. Did you find this not to be the case in your testing?

Reply:
I just added it without thinking about it, I will remove it.